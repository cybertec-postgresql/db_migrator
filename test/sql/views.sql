\i test/psql.sql

SELECT plan(2);

SELECT is(
    db_migrate_views(plugin => 'noop_migrator'),
    0,
    'Should create views without errors'
);

SELECT has_view('sch1', 'v1', 'View v1 should exist');

SELECT * FROM finish();