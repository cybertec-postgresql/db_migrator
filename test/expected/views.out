\i test/psql.sql
\unset ECHO
1..2
ok 1 - Should create views without errors
ok 2 - View v1 should exist
